openapi/components/schemas/Cashier/CashierStrategy.yaml

```
@@ -0,0 +1,35 @@
+type: object
+required:
+  - name
+  - filter
+  - amounts
+  - customAmount
+properties:
+  id:
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  name:
+    type: string
+    description: Name of the strategy.
+  filter:
+    type: string
+    description: |-
+      Filter that uses cashier request properties to determine if a cashier strategy is applicable for the request.
+      If this field is empty, the strategy is applicable for any request.
+    default: ''
+    example: 'cashierRequest.currency:USD'
+    allOf:
+      - $ref: ../FilterString.yaml
+  amounts:
+    $ref: ./CashierStrategyAmounts.yaml
+  customAmount:
+    description: |-
+      Custom amount restrictions.
+      If this value is `null`,  custom amounts are prohibited.
+    allOf:
+      - $ref: ./CashierStrategyCustomAmount.yaml
```

```suggestion
    $ref: ./CashierStrategyCustomAmount.yaml
```

I've had to make extensive amounts of suggestions on this PR where I should not have had to do that.

This is disappointing. You mark my comments as addressed, but they are not.